sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
> What happens if the user changes the tip amount after Payday 2 though?

I think that we should be able to isolate cases where this has happened.. And if we're looking to get this out the door before https://github.com/gratipay/gratipay.com/issues/3851 is done - the best way to go about this is reset the dues to $0 for only those users. For users who did *not* change the tip amount at all, the original [`UPDATE` statement](https://github.com/gratipay/gratipay.com/pull/3876#discussion-diff-51040424) should work just fine

> Agreed. How many users are affected if we go this route? I suspect it's not very many ...

In total - it'd be a lot of people (`649`, to be exact). But after isolating the ones that we can't properly repair without #3851, it might not be many... Time to run some queries..